assemblies/assembly_executing-processes/main.adoc

```diff
@@ -13,7 +13,7 @@ include::_artifacts/author-group.adoc[]
 As a business analyst or business rules developer, you can use {CENTRAL} to create forms for human tasks, providing a rich interface for collecting data. In this example, you will create a simple pizza order form that a customer (Bill) will complete and send to the pizza place. The pizza place employee (Katy) will process the order and send an order confirmation message containing the total cost for the order.
 
 .Prerequisites
-* {EAP_LONG} 7.2.0 is installed.For details,see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
+* {EAP_LONG} {EAP_VERSION} is installed. For details, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/installation_guide/[_Red Hat JBoss EAP 7.3.0 Installation Guide_].
```

Comment:
Still targeting EAP 7.2

Reply:
When published, this will update for all docs.